gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/apim/core/documentation/model/Page.java

```diff
@@ -54,15 +54,15 @@ public class Page {
     // Non-Folder attributes
     private String content;
     private boolean homepage;
+    private boolean excludedAccessControls;
+    private Set<AccessControl> accessControls;
```

Comment:
Note: I think you could apply access control on folders too

Reply:
Ok I'll move it to the general attributes. The use case code applies to folders as well by default 👍